addon/templates/frost-list.hbs

```diff
@@ -0,0 +1,36 @@
+{{yield}}
+
+<div class="header">
+  {{#yield-slot 'header' (block-params (hash
+  expansion=(component 'frost-list-toggle' hook=(concat hook '-toggle') showDetail=showDetail onClick=(action (mut showDetail) value="target.value"))
```

Comment:
We don't want to bind mut as a default action here - we should fire out onExpand / onCollapse and let that data trickle down from the controller